sql/main_summary_v5.sql

```diff
@@ -0,0 +1,357 @@
+  SAFE_CAST(environment.system.os.windows_ubr AS INT64) AS windows_ubr,
+
+  -- Note: Windows only!
+  SAFE_CAST(environment.system.os.install_year AS INT64) AS install_year,
+  environment.system.is_wow64,
+
+  SAFE_CAST(environment.system.memory_mb AS INT64) AS memory_mb,
+
+  environment.system.cpu.count AS cpu_count,
+  environment.system.cpu.cores AS cpu_cores,
+  environment.system.cpu.vendor AS cpu_vendor,
+  environment.system.cpu.family AS cpu_family,
+  environment.system.cpu.model AS cpu_model,
+  environment.system.cpu.stepping AS cpu_stepping,
+  SAFE_CAST(environment.system.cpu.l2cache_kb AS INT64) AS cpu_l2_cache_kb,
+  SAFE_CAST(environment.system.cpu.l3cache_kb AS INT64) AS cpu_l3_cache_kb,
+  SAFE_CAST(environment.system.cpu.speed_m_hz AS INT64) AS cpu_speed_mhz,
+
+  environment.system.gfx.features.d3d11.status AS gfx_features_d3d11_status,
+  environment.system.gfx.features.d2d.status AS gfx_features_d2d_status,
+  environment.system.gfx.features.gpu_process.status AS gfx_features_gpu_process_status,
+  environment.system.gfx.features.advanced_layers.status AS gfx_features_advanced_layers_status,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.gfx.features.wr_qualified.status") AS gfx_features_wrqualified_status,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.gfx.features.webrender.status") AS gfx_features_webrender_status,
+
+  -- Note: Windows only!
+  environment.system.hdd.profile.type AS hdd_profile_type,
+  environment.system.hdd.binary.type AS hdd_binary_type,
+  environment.system.hdd.system.type AS hdd_system_type,
+
+  environment.system.apple_model_id,
+
+  -- Note: Windows only!
+  environment.system.sec.antivirus,
+  environment.system.sec.antispyware,
+  environment.system.sec.firewall,
+
+  -- TODO: use proper 'date' type for date columns.
+  SAFE_CAST(environment.profile.creation_date AS INT64) AS profile_creation_date,
+  SAFE_CAST(environment.profile.reset_date AS INT64) AS profile_reset_date,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.previous_build_id") AS previous_build_id,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.session_id") AS session_id,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.subsession_id") AS subsession_id,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.previous_subsession_id") AS previous_subsession_id,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.session_start_date") AS session_start_date,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.session_length") AS INT64) AS session_length,
+  payload.info.subsession_length,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.subsession_counter") AS INT64) AS subsession_counter,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.info.profile_subsession_counter") AS INT64) AS profile_subsession_counter,
```

Comment:
`payload.info` looks like a good section to add to the main ping schema, it's currently bare.